### PR TITLE
Sync various sections

### PR DESCRIPTION
**Pull request by saundefined**

### CODE REVIEW

File: reference/opcache/ini.xml [hunk 1]
```diff
@@ -1078,6 +1080,10 @@
       Option de débogage qui désactive la compilation JIT après la compilation d'un certain nombre
       de fonctions.
       Ceci peut être utile pour bisect la source d'une mauvaise compilation JIT.
+      Remarque : cette option ne fonctionne que lorsque le déclencheur JIT est défini
+      sur 0 (compilation au chargement du script) ou 1 (compilation à la première exécution),
```
Comment (Girgias) [inline reference/opcache/ini.xml:1084]:
```suggestion
      à 0 (compilation au chargement du script) ou 1 (compilation à la première exécution),
```

File: reference/opcache/ini.xml [hunk 2]
```diff
@@ -1138,6 +1144,9 @@
     <listitem>
      <para>
       Après combien d'itération une boucle est considéré chaude.
+      La plage de valeurs valides est <code>[0,255]</code> ;
```
Comment (Girgias) [inline reference/opcache/ini.xml:1147]:
```suggestion
      L'intervalle de valeurs valides est <code>[0,255]</code> ;
```
Is better in this context (and for all the other cases in this file (sorry can't really do the edit myself atm))

**Review by Girgias — CHANGES_REQUESTED** [review]
Other than the slight issue with the Opcache INI file, LGTM. :)

**Comment by saundefined** [comment]
Thanks as always for the review ;)

**Pull request merged**